### PR TITLE
Replaced idam searchUsers api with getUserByUserId

### PR DESCRIPTION
**Pull request by smathangi**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ACA-95

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by tomxelliott — APPROVED** [review]
Looks good to me 👍

**Review by sullivangareth — APPROVED** [review]

**Review by MSancaktutar — APPROVED** [review]
LGTM

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -43,7 +43,7 @@ idam:
   client:
     id: aac_manage_case_assignment
     secret: ${IDAM_CLIENT_SECRET:AAAAAAAAAAAAAAAA}
-    scope: openid roles profile search-user
+    scope: openid roles profile manage-user
```
Comment (mattnayler) [inline src/main/resources/application.yaml:46]: Please also update: [/aca-docker/bin/add-idam-clients.sh](https://github.com/hmcts/aac-manage-case-assignment/blob/idam_search_fix/aca-docker/bin/add-idam-clients.sh#L8)

**Review by mattnayler — CHANGES_REQUESTED** [review]
LGTM:  one additional change requested + [LLD External Interfaces](https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1373537536#APIOperation:AssignAccesswithinOrganisation-2.3-ExternalInterfaces)

**Pull request merged**